File: Q6.java
import java.util.Scanner;

public class Q6 
{
    public static Scanner leia = new Scanner(System.in);

    public static void main(String args[]) 
    {
        int anos, meses, dias, dias_vividos;
        System.out.print("Digite a sua idade: ");
        anos = leia.nextInt();
        System.out.print("Digite quantos meses se passaram desde seu aniversário: ");
        meses = leia.nextInt();
        System.out.print("Digite quantos dias você viveu no mês atual: ");
        dias = leia.nextInt();
        dias_vividos = anos * 365 + meses * 30 + dias;
        System.out.println("Você viveu um total de " + dias_vividos + " dias.");
    }
}

File: Q6.cpp
#include <iostream>

using namespace std;

int main()
{
    int anos, meses, dias, dias_vividos;
    cout << "Digite a sua idade: ";
    cin >> anos;
    cout << "Digite quantos meses se passaram desde seu aniversário: ";
    cin >> meses;
    cout << "Digite quantos dias você viveu no mês atual: ";
    cin >> dias;
    dias_vividos = anos * 365 + meses * 30 + dias;
    cout << "Você viveu um total de " << dias_vividos << " dias." << endl;
    return 0;
}